src/types/types.ts

```diff
@@ -813,6 +830,86 @@ export interface GetFloorPriceResponse {
   readonly looksRare: FloorPriceMarketplace | FloorPriceError;
 }
 
+/**
+ * Optional parameters object for the the {@link getContractsForOwner} method
+ *
+ * @public
+ */
+export interface GetContractsForOwnerOptions {
+  /** Key for pagination to use to fetch results from the next page if available. */
+  pageKey?: string;
+
+  /**
+   * Optional list of filters applied to the query. NFTs that match one or more
+   * of these filters are included in the response. May not be used in
+   * conjunction with {@link excludeFilters}.
+   */
+  includeFilters?: NftFilters[];
+
+  /**
+   * Optional list of filters applied to the query. NFTs that match one or more
+   * of these filters are excluded from the response. May not be used in
+   * conjunction with {@link includeFilters}
+   */
+  excludeFilters?: NftFilters[];
+}
+
+/**
+ * The response for the {@link NftNamespace.getContractsForOwner} method.
+ *
+ * @public
+ */
+export interface GetContractsForOwnerResponse {
+  /** The list of contracts, that match the query, held by the given address. */
+  contracts: ContractForOwner[];
+
+  /** Key for pagination to use to fetch results from the next page if available. */
+  pageKey?: string;
+
+  /** Total number of NFT contracts held by the given address. */
+  totalCount: number;
+}
+
+/** Represents a single NFT contract data in the {@link GetContractsForOwnerResponse}. */
+export interface ContractForOwner {
```

Comment:
Instead of creating a completely new interface, can we extend the additional fields on top of the existing `NftContract` object?

Reply:
Sure, I'll update it tomorrow. Thank you for the review. It is an excellent suggestion 🔥

Reply:
I've just made the requested changes. I've noticed that there was a failing test for the NFT API and fixed it. Also, there are 2 integration tests that have `console.log` I don't know if that is something that should be removed.
![image](https://user-images.githubusercontent.com/42520800/207399090-f6f169fc-2112-445a-9464-1a13b170d3b0.png)
![image](https://user-images.githubusercontent.com/42520800/207399138-d744555c-8423-4846-b4af-427e9f01d614.png)